src/utils/importPaths.ts

```diff
@@ -96,10 +96,13 @@ export const WM_WRITE_FELT: [string[], string] = [[...WARPLIB_MEMORY], 'wm_write
 
 //------------------------------------------------------
 
+export const ARRAY: [string[], string] = [['array'], 'Array'];
 export const ARRAY_TRAIT: [string[], string] = [['array'], 'ArrayTrait'];
 export const U256_FROM_FELTS: [string[], string] = [['warplib', 'integer'], 'u256_from_felts'];
 export const GET_U128: [string[], string] = [['integer'], 'u128_from_felt'];
 export const U128_TO_FELT: [string[], string] = [['integer'], 'u128_to_felt'];
+export const GET_U32: [string[], string] = [['integer'], 'u32_from_felt'];
+export const U32_TO_FELT: [string[], string] = [['integer'], 'u32_to_felt'];
```

Comment:
nitpick: rename `GET_U32` to `U32_FROM_FELT`. 

I notice that we have a `GET_U128` as well. In case of accepting the suggestion, that should be renamed as well